internal/pkg/service/buffer/store/configstore/configstore.go

```diff
@@ -49,6 +53,15 @@ func MappingKey(projectID int, receiverID string, exportID string, revisionID in
 	return fmt.Sprintf("config/mapping/revision/%d/%s/%s/%08d", projectID, receiverID, exportID, revisionID)
 }
 
+func RecordKey(projectID int, receiverID string, exportID string, fileID string, sliceID string) string {
+	recordID := FormatTimeForKey(time.Now()) + "_" + idgenerator.Random(5)
+	return fmt.Sprintf("record/%d/%s/%s/%s/%s/%s", projectID, receiverID, exportID, fileID, sliceID, recordID)
```

Comment:
Chyba mi tu time mock, bez toho sa to bude blbo testovat.
Jeden zaznam sa da, tak ako si to spravil ty, ale nejake zlozitejsie operacie tazko.
Mozno by ten `Record(Key)` mohol mat field `time`?

Reply:
Tomu asi taky nerozumím. 😅 Mám otestovaný že prefix vypadá tak jak očekáváme, zbytek je asi jedno? Respektive měl bych přidat test, že když vytvořím dva za sebou, tak nebudou stejný. To udělám. Jinak bych to asi potřeboval víc vysvětlit. 🙂 Ten time mock, případně tu poslední větu.

Reply:
- nevies v teste ovplyvnit aky cas sa tam generuje, pouzivaju sa sytemove hodiny
- bolo by dobre ak by si si vedel cas namockovat
- hlavne pre zlozitejsie operacie
- https://stackoverflow.com/a/18970352

Reply:
A ten `realClock` má být další parametr `RecordKey`, nebo jak je to myšleno? (BTW. nevím úplně co myslíš těma složitějšíma operacema, ale věřím ti. 😄)

Reply:
mozeme zavolat, alebo to nechat tak .... no ja budem testovat export viacerych rows do file storage a potrebujem tam vidiet tie casy, a preto ich potrebujem staticke, ... ked tam tam `/*`, tak nezistim ci tam nejaky datum je alebo nie.

Reply:
ale kludne to nechaj tak. ..... najjednoduchsie bude dat `receivedAt time.Time` do `Record` struktury

Reply:
https://github.com/keboola/keboola-as-code/pull/1007/commits/a90aeffba2e94f33aca963349fa9609e7b64099a :+1:

Reply:
Už mi to `receivedAt` a `RecordKey.String()` konečně docvakly. 😄